tests/ops/op_math/test_controlled.py

```diff
@@ -1639,6 +1641,47 @@ def test_no_ctrl_custom_operators_control_values(self, op_cls, custom_op_cls):
         assert not isinstance(ctrl_op, custom_op_cls)
         assert qml.equal(ctrl_op, expected)
 
+    def test_ctrl_PauliX_one_wire(self):
+        """Tests the type of Operator created when calling ctrl(PauliX)"""
+        with qml.queuing.AnnotatedQueue() as q:
+            op = qml.ctrl(qml.PauliX(1), 0)
+
+        expected = qml.CNOT([0, 1])
+        assert len(q) == 1
+        assert qml.equal(op, expected)
+        assert qml.equal(q.queue[0], expected)
```

Comment:
I wonder if we can condense these new tests for one, two and three wires using the ones right above. The `custom_controlled_ops` list contains the operations for which we support a custom controlled version of themselves in `qml.ctrl`. We could expand the tuples to include the number of wires (just like we did inside `qml.ctrl`) and adapt the 3 tests above to account for that. I think this'll make testing future custom controlled classes easier with less code (sorry more things! 😄)

At least, I would include `qml.PauliX` in the list and skip the single control wire test.